src/Makefile.am

```diff
@@ -471,6 +471,10 @@ if HARDEN
 	$(AM_V_at) READELF=$(READELF) OBJDUMP=$(OBJDUMP) $(top_srcdir)/contrib/devtools/security-check.py < $(bin_PROGRAMS)
 endif
 
+check-formatting:
+	@echo "Checking source formatting style..."
+	(cd $(top_srcdir)/src; ../contrib/devtools/clang-format.py check clang-format-3.8 `cat ./.formatted-files`)
```

Comment:
Yes I thought so too... and got pretty far into getting it from a ppa.  But then I loaded up my ubuntu 14.04.5 and lo and behold it is actually available.  I think it was added as part of the update...